README.md

```diff
@@ -1,5 +1,17 @@
 # genome-nexus-java-api-client
 Java API Client for [Genome Nexus](https://www.genomenexus.org)
 
+To generate the api client, run:
```

Comment:
"To generate the API client, based on the currently deployed genome nexus production server, run:"